src/components/common/Carousel/Carousel.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { Carousel } from './Carousel';
+
+export default {
+  component: Carousel,
+  title: 'common/Carousel',
+} as Meta;
+
+const Template: Story = (args) => (
+  <Carousel {...args}>
+    {[...Array(7)].map((_, index) => (
+      // eslint-disable-next-line react/no-array-index-key
+      <div key={index} css={{ width: 300, height: 300, border: '1px solid' }} />
+    ))}
```

Comment:
[imo]
`disable` しなくてもこれで良いかなと思いました。

```suggestion
    {[...Array(7)].map((num) => (
      <div key={num} css={{ width: 300, height: 300, border: '1px solid' }} />
    ))}
```